bridge-token-factory/src/lib.rs

```diff
@@ -381,12 +595,18 @@ impl BridgeTokenFactory {
                     ext_bridge_token::ext(self.get_bridge_token_account_id(token))
                         .with_static_gas(FT_TRANSFER_CALL_GAS)
                         .with_attached_deposit(1)
-                        .ft_transfer_call(target, amount.into(), None, message),
+                        .ft_transfer_call(target, amount_to_transfer.into(), None, message),
                 ),
-            None => ext_bridge_token::ext(self.get_bridge_token_account_id(token))
+            None => ext_bridge_token::ext(self.get_bridge_token_account_id(token.clone()))
                 .with_static_gas(MINT_GAS)
                 .with_attached_deposit(env::attached_deposit() - required_deposit)
-                .mint(target, amount.into()),
+                .mint(target, amount_to_transfer.into())
+                .then(
+                    ext_bridge_token::ext(self.get_bridge_token_account_id(token))
+                        .with_static_gas(MINT_GAS)
+                        .with_attached_deposit(env::attached_deposit() - required_deposit)
```

Comment:
Not sure about that place... we have the same attached deposit to times in a raw. Don't we already spend it on the first call? Really not sure, how it works.

Reply:
feff748af2b5e00f67d773343188b219d4317590

Reply:
It was also changed in 1b1d3fd1170eda4a4b7307c93bfda29533cd9f12.
So now for a deposit, there are no extra requirements for the storage deposit.
But it is not possible to attach a `deposit` for the withdrawal, so the storage deposit should be done while setting a fee for the token by `set_withdraw_fee`. 
Also, note that there is no storage deposit on calling `set_withdraw_fee_per_silo` because the token address could be none, so it is not possible to cover the storage deposit for all tokens when the fee is configured for all tokens in the silo. This case is fixed by adding a storage deposit in the `BridgeToken::new()` on deploying new tokens, but there is no solution for already deployed tokens, so this process should be done manually on migration.